README.md

```diff
@@ -40,6 +40,7 @@ To run the app, follow these steps.
   gulp watch
   ```
 7. Browse to [http://localhost:9000](http://localhost:9000) to see the app. You can make changes in the code found under `src` and the browser should auto-refresh itself as you save files.
+> The Skeleton App uses [BrowserSync](http://www.browsersync.io/) for automated page refreshes on code/markup changes concurrently accross multiple browsers. If you prefer to disable the mirroring feature set the [ghostMode option](http://www.browsersync.io/docs/options/#option-ghostMode) to false
```

Comment:
I still don't have a clear idea why this is set as the default behaviour in the skeleton. Is it to speed up manual cross-browser testing? Maybe a mention of the unquestionable usefulness of that could be made here.